{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2014-3608",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "vendor_name": "openstack",
            "product": {
              "product_data": [
                {
                  "product_name": "compute",
                  "version": {
                    "version_data": [
                      {
                        "version_value": "2014.1"
                      },
                      {
                        "version_value": "2014.1.1"
                      },
                      {
                        "version_value": "2014.1.2"
                      }
                    ]
                  }
                }
              ]
            }
          }
        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-399"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://rhn.redhat.com/errata/RHSA-2014-1781.html"
        },
        {
          "url": "http://rhn.redhat.com/errata/RHSA-2014-1782.html"
        },
        {
          "url": "http://seclists.org/oss-sec/2014/q4/65"
        },
        {
          "url": "http://www.securityfocus.com/bid/70220"
        },
        {
          "url": "https://bugs.launchpad.net/nova/+bug/1338830"
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "The VMWare driver in OpenStack Compute (Nova) before 2014.1.3 allows remote authenticated users to bypass the quota limit and cause a denial of service (resource consumption) by putting the VM into the rescue state, suspending it, which puts into an ERROR state, and then deleting the image.  NOTE: this vulnerability exists because of an incomplete fix for CVE-2014-2573."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "operator": "OR",
        "cpe": [
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:openstack:compute:2014.1",
            "cpe23Uri": "cpe:2.3:a:openstack:compute:2014.1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:openstack:compute:2014.1.1",
            "cpe23Uri": "cpe:2.3:a:openstack:compute:2014.1.1:*:*:*:*:*:*:*"
          },
          {
            "vulnerable": true,
            "cpe22Uri": "cpe:/a:openstack:compute",
            "cpe23Uri": "cpe:2.3:a:openstack:compute:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "2014.1.2"
          }
        ]
      }
    ]
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "version": "2.0",
        "vectorString": "(AV:A/AC:L/Au:S/C:N/I:N/A:P)",
        "accessVector": "ADJACENT_NETWORK",
        "accessComplexity": "LOW",
        "authentication": "SINGLE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 2.7
      },
      "severity": "LOW",
      "exploitabilityScore": 5.1,
      "impactScore": 2.9,
      "obtainAllPrivilege": false,
      "obtainUserPrivilege": false,
      "obtainOtherPrivilege": false,
      "userInteractionRequired": false
    }
  },
  "publishedDate": "2014-10-06T14:55Z",
  "lastModifiedDate": "2016-11-28T19:11Z"
}